Mention that the A.js snippet has version history available on GitHub (#4072)
segmentio/segment-docs: src/getting-started/02-simple-install.md

> note ""
> **Note:** If you only want the most basic Google Analytics setup you can stop reading right now. You're done! Just toggle on Google Analytics from the Segment App.

> info ""

> > Our snippet version history is open-source and can be [viewed on GitHub](https://github.com/segmentio/snippet/blob/master/History.md). It is always recommended to use the latest snippet version whenever possible.

Reviewer: ```suggestion
> The Segment snippet version history is open-source and can be [viewed on GitHub](https://github.com/segmentio/snippet/blob/master/History.md). It is always recommended to use the latest snippet version whenever possible.
```

> > The Segment snippet version history is open-source and can be [viewed on GitHub](https://github.com/segmentio/snippet/blob/master/History.md). It is always recommended to use the latest snippet version whenever possible.

Reviewer: ```suggestion
> The Segment snippet version history available on [GitHub](https://github.com/segmentio/snippet/blob/master/History.md){:target="_blank"}. Segment recommends that you use the latest snippet version whenever possible.
```
